File: co.edu.uniandes.fixml.toCSharp/test/text1.xml/Instrmt.cs
namespace text1.xml;



// Class for tag Instrmt
class Instrmt {

	string Sym;
	string ID;
	string IDSrc;
	
	
	// default constructor
	
	Instrmt ( ) {   
		this.Sym = "IBM";
		this.ID = "459200101";
		this.IDSrc = "1";
		
		
	}
	
	Instrmt ( String Sym, String ID, String IDSrc ) {
		this.Sym = Sym;
		this.ID = ID;
		this.IDSrc = IDSrc;
		
	}
	
	
} 

File: co.edu.uniandes.fixml.toC++/test/test1.xml/Instrmt.cpp
#include <iostream> 

using namespace std;

class Instrmt {

	private:
  	string ID_;
  	string IDSrc_;
  	string Sym_;
 	
 	public: 

  	// default constructor
	Instrmt () {
		ID_ = "459200101";
		IDSrc_ = "1";
		Sym_ = "IBM";
  	
  	}
  
  	// by parameters
  	Instrmt (		string ID_p, string IDSrc_p, string Sym_p){
	  
		ID_ = ID_p;
		IDSrc_ = IDSrc_p;
		Sym_ = Sym_p;
	  
  	}
};